cssom-1/Overview.bs

```diff
@@ -1078,19 +1078,26 @@ CSS Style Sheet Collections {#css-style-sheet-collections}
 Below various new concepts are defined that are associated with each
 {{DocumentOrShadowRoot}} object.
 
-Each {{DocumentOrShadowRoot}} has an associated list of zero or more
-<a>CSS style sheets</a>, named the
+Each {{DocumentOrShadowRoot}} has an associated list of zero or more <a>CSS style sheets</a>, named the
 <dfn export for=DocumentOrShadowRoot>document or shadow root CSS style sheets</dfn>. This is
-an ordered list that contains all
-<a>CSS style sheets</a> associated with the
-{{DocumentOrShadowRoot}}, in
-<a>tree order</a>, with
-<a>CSS style sheets</a> created from HTTP
-<code>Link</code> headers first, if any, in header
-order.
-
-To <dfn export>create a CSS style sheet</dfn>, run these
-steps:
+an ordered list that contains:
+<ol>
+  <li>Any <a>CSS style sheets</a> created from HTTP <code>Link</code> headers, in header order</li>
+  <li>Any <a>CSS style sheets</a> associated with the {{DocumentOrShadowRoot}}, in <a>tree order</a></li>
+</ol>
+
+Each {{DocumentOrShadowRoot}} has an associated list of zero or more <a>CSS style sheets</a>, named the
+<dfn export for=DocumentOrShadowRoot>final CSS style sheets</dfn>. This is
+an ordered list that contains:
+<ol>
+  <li>Any <a>CSS style sheets</a> created from HTTP <code>Link</code> headers, in header order</li>
+  <li>Any <a>CSS style sheets</a> associated with the {{DocumentOrShadowRoot}}, in <a>tree order</a></li>
+  <li>The contents of {{DocumentOrShadowRoot}}'s <a href="#dom-documentorshadowroot-adoptedstylesheets">adoptedStyleSheets</a>'
```

Comment:
This is an internal reference, and I couldn't get this one working either. Could someone suggest the correct syntax to link from here to this definition below?

```html
<dfn attribute for="DocumentOrShadowRoot"><code>adoptedStyleSheets</code></dfn>
```

Reply:
`{{DocumentOrShadowRoot/adoptedStyleSheets}}` should do it.

Reply:
Yep, that's the magic. Thanks!